ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:isCheckedOut="{Boolean}false"
+    jcr:primaryType="cq:WorkflowModel"
+    jcr:uuid="72e1b6ff-d73e-4a66-b6c6-354718451e54"
+    sling:resourceType="cq/workflow/components/model"
+    description="No Description"
+    title="Content Approval Workflow">
```

Comment:
@nitheshkumar-hcl  updated group name and append HCLecomm in name.

Reply:
updated the workflow as per the review comments